libs/common/src/test-helpers/schemas/with-entra-id-user-schema.tests.ts

```diff
@@ -18,11 +17,11 @@ export function aValidEntraIdUser(): EntraIdUser {
     verified_secondary_email: ['a-secondary-email'],
     given_name: 'a-given-name',
     family_name: 'a-family-name',
-    groups: [ENTRA_ID_USER_GROUP.AZURE_SSO_GROUP_BUSINESS_SUPPORT],
+    roles: ['BUSINESS_SUPPORT'],
```

Comment:
constant?

Reply:
No need in TS as it's typed so will kick up a fuss if it's not correct -- but I've updated